### ISSUE TITLE
With findbugs plugin 3.8 , Sonar java 5.7 and SonarQube LTS 6.7.5 , analysis is failing as below

### ISSUE THREAD

**Issue by gopal9590** [spotbugs/sonar-findbugs#212]
Hi ,

Thank you for findbugs plugin.

We recently upgraded from version 5.5 to LTS 6.7.5 , findbugs plugin also got upgraded from 3.3 to 3.8

After the upgrade , sonar analysis is failing with below error , can you please help me know what could be the issue or if this is already known bug:

Thank you so much !
```
Error processing opcode checkcast @ 69 in akka.actor.SupervisorStrategy$.$anonfun$sort$1 : (Lscala.collection.mutable.ArrayBuffer;Lscala.Tuple2;)Lscala.collection.mutable.ArrayBuffer;
    java.lang.IllegalArgumentException: Unknown signature [Ljava/lang/Object; for number 1
      At edu.umd.cs.findbugs.OpcodeStack$Item.<init>(OpcodeStack.java:671)
      At edu.umd.cs.findbugs.OpcodeStack.sawOpcode(OpcodeStack.java:1725)
      At edu.umd.cs.findbugs.bcel.OpcodeStackDetector.afterOpcode(OpcodeStackDetector.java:83)
      At edu.umd.cs.findbugs.visitclass.DismantleBytecode.visit(DismantleBytecode.java:884)
      At edu.umd.cs.findbugs.detect.FindNoSideEffectMethods.visit(FindNoSideEffectMethods.java:492)
      At edu.umd.cs.findbugs.visitclass.BetterVisitor.visitCode(BetterVisitor.java:218)
      At edu.umd.cs.findbugs.visitclass.PreorderVisitor.visitCode(PreorderVisitor.java:243)
      At edu.umd.cs.findbugs.bcel.OpcodeStackDetector.visitCode(OpcodeStackDetector.java:65)
      At org.apache.bcel.classfile.Code.accept(Code.java:132)
      At edu.umd.cs.findbugs.visitclass.PreorderVisitor.doVisitMethod(PreorderVisitor.java:315)
      At edu.umd.cs.findbugs.visitclass.PreorderVisitor.visitJavaClass(PreorderVisitor.java:403)
      At org.apache.bcel.classfile.JavaClass.accept(JavaClass.java:213)
      At edu.umd.cs.findbugs.BytecodeScanningDetector.visitClassContext(BytecodeScanningDetector.java:38)
      At edu.umd.cs.findbugs.DetectorToDetector2Adapter.visitClass(DetectorToDetector2Adapter.java:76)
      At edu.umd.cs.findbugs.FindBugs2.analyzeApplication(FindBugs2.java:1080)
      At edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:281)
      At org.sonar.plugins.findbugs.FindbugsExecutor$FindbugsTask.call(FindbugsExecutor.java:235)
      At java.util.concurrent.FutureTask.run(FutureTask.java:266)
      At java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
      At java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
      At java.lang.Thread.run(Thread.java:748)
The following classes needed for analysis were missing:
  test
  accept
  apply
  compare
````

**Comment by h3xstream** [comment]
This looks like a spotbugs issue : https://github.com/spotbugs/spotbugs/issues/715, https://github.com/spotbugs/spotbugs/issues/758

**Comment by venkatmeena** [comment]
Hi,

Greetings !!

We recently upgraded from Sonarqube v5.5 to 6.7.5 LTS , SonarJava 5.6.1 and findbugs plugin was upgraded to 3.7.
We are facing the same issue with findbugs and analysis is taking too much time to complete.Is this bugs exists in v3.7?

Thanks you !

**Comment by swetha1604** [comment]
Hi,

Greetings!!

We recently upgraded sonarqube from version 5.5 to LTS 6.7.5 , findbugs plugin was upgraded from 3.3 to 3.7.
We facing the same issue with v3.7.Is there any fix or workaround for this?

Thank you!!

**Comment by KengoTODA** [comment]
solved by 3.9
